connectivity/suite.go

```diff
@@ -5,6 +5,9 @@ package connectivity
 
 import (
 	"context"
+	"errors"
+
+	"github.com/hashicorp/go-multierror"
```

Comment:
maybe use https://pkg.go.dev/errors#Unwrap instead? you just need to accumulate errors and unwrap right? @tklauser what do you think?

Reply:
Ack, using `%w` and/or `errors.Unwrap` would be preferable.

Reply:
Not sure here... `go-multierror` allows to [run many goroutines and collect all the errors](https://github.com/cilium/cilium-cli/pull/2496/files#diff-ab78417b50a516eda19c907a86d67689ea4deb1708e96e6efab60bcce67765b4R49-R56).
Am I missing something?

Reply:
no you are not missing anything, it's just a tradeoff between writing our own code with standard libraries vs using third party libraries. i think in this case it's reasonable to write a bit more code to collect errors from goroutines and avoid depending on another library.

Reply:
I've implemented [MultiError runner](https://github.com/cilium/cilium-cli/pull/2496/files#diff-9f58834ede9ee2a7744d0a7fb594cd2b764629ce312f4898979924bd74cecf81R14) component to replace HashiCorp lib.